suse-module-tools.spec

```diff
@@ -99,6 +101,16 @@ This package contains the legacy "weak-modules" script for kernel
 module package (KMP) support. It was replaced by "weak-modules2" in
 SLE 11 and later. It is still used by the DKMS module packaging framework.
 
+%if 0%{?suse_version} >= 1550
+%package -n suse-kernel-rpm-scriptlets
```

Comment:
Perhaps you should choose a package name that differs from the resolvable name?

Reply:
Any suggestions?

Reply:
According to your own choice of words, maybe `suse-legacy-kernel-rpm-scriptlets`? But that's a bit lengthy... 
I really don't care much. But I think it would be good to separate package name and resolvable, in particular because you intend to exchange the package for something else.